lib/configs/angular.js

```diff
@@ -47,7 +47,6 @@ module.exports = {
         }],
         'no-with': 0,
         'no-mixed-spaces-and-tabs': 2,
-        'no-multiple-empty-lines': 2,
```

Comment:
It just moved down in the file with an additional rule about EoF empty lines. This rule is still three